### PR TITLE
Adding basic UserListServlet for community page

### PR DESCRIPTION
**Pull request by pkhodadoustan**
This Is a basic UserListServlet java class. At this point it has a doGet() method that prints a hard coded message.

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/UserListServlet.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ */
+@WebServlet("/user-list")
+public class UserListServlet extends HttpServlet{
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+
+    response.getOutputStream().println("this will be my user list");
+  }
+
```
Comment (ginellegaisano) [inline src/main/java/com/google/codeu/servlets/UserListServlet.java]: nit: remove newline whitespace.

This is a style suggestion - usually we don't have a newline between the last function and the containing class.

Comment (pkhodadoustan) [reply src/main/java/com/google/codeu/servlets/UserListServlet.java]: Hi Ginelle, 
Thank you for your suggestion. I checked the files and deleted the newline :)

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -79,4 +81,16 @@ public void storeMessage(Message message) {
...
+	{
+		Set<String> users = new HashSet<>();
+		Query query = new Query("Message");
+		PreparedQuery results = datastore.prepare(query);
+		for(Entity entity: results.asIterable())
+		{
+			users.add((String)entity.getProperty("user"));
+		}
+		return users;
+	}
```
Comment (ginellegaisano) [inline src/main/java/com/google/codeu/data/Datastore.java:128]: nit: unindent to match other functions.

File: src/main/webapp/js/community-page-loader.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+      /** Fetches users and adds them to the page. */
+      function fetchUserList(){
```
Comment (ginellegaisano) [inline src/main/webapp/js/community-page-loader.js]: nit: there should be space between () and {. Also fix below on line 21.

File: src/main/java/com/google/codeu/servlets/UserListServlet.java [hunk 4]
```diff
@@ -16,31 +16,23 @@
...
-//              response.getOutputStream().println(user);
-//          }
-            response.setContentType("application/json");
-            Gson gson = new Gson();
-            String json = gson.toJson(users);
-            response.getOutputStream().println(json);
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    //          response.getOutputStream().println("this will be my user list");
```
Comment (ginellegaisano) [inline src/main/java/com/google/codeu/servlets/UserListServlet.java]: nit: delete commented out line.

**Review by ginellegaisano — APPROVED** [review]

**Pull request merged**